### PR TITLE
Heading color in Video section

### PR DESCRIPTION
**Pull request by PrudinnikDmitriy**

### CODE REVIEW

File: src/sections/home-video.liquid [hunk 1]
```diff
@@ -33,10 +33,24 @@
   </div>
 </div>
 
+<style>
+  {%- if section.setting.header_color.alpha != 0 and section.settings.header_color != blank -%}
+    .header-color {
```
Comment (yevheniitsuryk) [inline src/sections/home-video.liquid:38]: @PrudinnikDmitriy did you check header_color when there are two sections on the page?

File: src/sections/home-video.liquid [hunk 2]
```diff
@@ -33,10 +33,24 @@
...
+      color: {{ section.settings.header_color }};
+  }
```
Comment (yevheniitsuryk) [inline src/sections/home-video.liquid:40]: @PrudinnikDmitriy formatting, pls fix

**Review by yevheniitsuryk — CHANGES_REQUESTED** [review]

File: src/sections/home-video.liquid (hunk 1 above)
Comment (PrudinnikDmitriy) [reply src/sections/home-video.liquid]: @yevheniitsuryk fixed

File: src/sections/home-video.liquid (hunk 2 above)
Comment (PrudinnikDmitriy) [reply src/sections/home-video.liquid]: @yevheniitsuryk fixed

File: src/sections/home-video.liquid [hunk 3]
```diff
@@ -33,10 +34,24 @@
   </div>
 </div>
 
+<style>
```
Comment (yevheniitsuryk) [inline src/sections/home-video.liquid:37]: @PrudinnikDmitriy if you are using one style in style tags, you may move condition out of style tag on the upper level

**Review by yevheniitsuryk — CHANGES_REQUESTED** [review]

**Review by AndriiHrohodza — APPROVED** [review]

File: src/sections/home-video.liquid (hunk 3 above)
Comment (PrudinnikDmitriy) [reply src/sections/home-video.liquid]: @yevheniitsuryk fixed

**Review by yevheniitsuryk — APPROVED** [review]

**Pull request merged**